Vizzuality/half-earth-v3
# Feature/challenges chart update

## Challenges chart update
### Description
- This PR migrates the challenges chart from PIXI back to d3 and addresses all the design changes proposed for the chart.
- A couple of new indicators for protected areas (proportion protected and protection needed) have been added to the x axis selector.
- The national priority layer (with the updated color ramp) has been added also.
- `framer-motion` has been added as a dependency to animate the chart
### Designs
[Designs](https://projects.invisionapp.com/d/main?origin=v7#/console/21108730/445055732/preview?scrollOffset=691)
### Testing instructions
`yarn install`
`yarn start`
Go into national report cards and check the Challenges view and play with the chart :)
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-274
https://half-earth-map.atlassian.net/browse/HE-229
https://half-earth-map.atlassian.net/browse/HE-278
